widget/embedded/src/containers/WidgetInfo/WidgetInfo.tsx

```diff
@@ -28,6 +29,8 @@ export function WidgetInfo(props: React.PropsWithChildren) {
   const swappers = useAppStore().swappers();
   const loadingStatus = useAppStore().fetchStatus;
   const resetLanguage = useLanguage().resetLanguage;
+  const getUnreadNotifications =
+    useNotificationStore().getUnreadNotifications();
```

Comment:
I believe "unreadNotifications" or "notifications" would be a more fitting choice.

Reply:
fixed.